x-pack/plugins/monitoring/public/application/pages/logstash/pipeline.tsx

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useContext, useState, useCallback, useEffect } from 'react';
+import { i18n } from '@kbn/i18n';
+import { find } from 'lodash';
+import { useKibana, useUiSetting } from '../../../../../../../src/plugins/kibana_react/public';
+import { GlobalStateContext } from '../../global_state_context';
+import { ComponentProps } from '../../route_init';
+// @ts-ignore
+import { List } from '../../../components/logstash/pipeline_viewer/models/list';
+// @ts-ignore
+import { PipelineViewer } from '../../../components/logstash/pipeline_viewer';
+// @ts-ignore
+import { Pipeline } from '../../../components/logstash/pipeline_viewer/models/pipeline';
+// @ts-ignore
+import { PipelineState } from '../../../components/logstash/pipeline_viewer/models/pipeline_state';
+// @ts-ignore
+import { vertexFactory } from '../../../components/logstash/pipeline_viewer/models/graph/vertex_factory';
+import { LogstashTemplate } from './logstash_template';
+import { useTable } from '../../hooks/use_table';
+import moment from 'moment';
+import { ExternalConfigContext } from '../../external_config_context';
+import { formatTimestampToDuration } from '../../../../common';
+import { CALCULATE_DURATION_SINCE } from '../../../../common/constants';
+import { useRouteMatch } from 'react-router-dom';
+
+export const LogStashPipelinePage: React.FC<ComponentProps> = ({ clusters }) => {
```

Comment:
the input/output click behavior doesn't seem to be working here. Should be a sidebar that slides out.